vit-servicing-station-lib/src/v0/context.rs

```diff
@@ -50,11 +54,16 @@ pub mod test {
         let db_url = format!("file:{}?mode=memory&cache=shared", name);
         let pool = db::load_db_connection_pool(&db_url).unwrap();
         let block0: Vec<u8> = vec![1, 2, 3, 4, 5];
-        Arc::new(RwLock::new(Context::new(pool, "", block0)))
+        Arc::new(RwLock::new(Context::new(
+            pool,
+            "",
+            block0,
+            "2.0".to_string(),
+        )))
     }
 
     pub fn new_test_shared_context(db_url: &str, block0_path: &str) -> SharedContext {
         let pool = db::load_db_connection_pool(db_url).unwrap();
-        new_shared_context(pool, block0_path)
+        new_shared_context(pool, block0_path, "2.0".to_string())
```

Comment:
We definitely need to have an agreement about versioning the VITss API and VITss as well (i.e. we will implement frequent tagging in CI for VITss repo), so there will be no need to hardcode the versioning in tests in near perspective.